src/text.c

```diff
@@ -20,39 +23,358 @@ extern u8* UnkTextUtil_GetPtrI(u8 a1);
 
 TextFlags gTextFlags;
 
-extern const u8 gDownArrowTiles[];
-extern const u8 gDarkDownArrowTiles[];
-extern const u8 gUnusedFRLGBlankedDownArrow[];
-extern const u8 gUnusedFRLGDownArrow[];
-extern const u8 gDownArrowYCoords[];
-extern const u8 gWindowVerticalScrollSpeeds[];
+const u8 gDownArrowTiles[]         = INCBIN_U8("data/graphics/fonts/down_arrow.4bpp");
+const u8 gDarkDownArrowTiles[]     = INCBIN_U8("data/graphics/fonts/down_arrow_RS.4bpp");
+const u8 gTinyArrowTiles[]         = INCBIN_U8("data/graphics/fonts/down_arrow_2.4bpp");
+const u8 gTinyDarkDownArrowTiles[] = INCBIN_U8("data/graphics/fonts/down_arrow_RS_2.4bpp");
+const u8 gDoubleArrowTiles[]       = INCBIN_U8("data/graphics/fonts/down_arrow_3.4bpp");
 
-extern const struct GlyphWidthFunc gGlyphWidthFuncs[];
+const u8 gDownArrowYCoords[]           = { 0x0, 0x10, 0x20, 0x10 };
+const u8 gWindowVerticalScrollSpeeds[] = { 0x1, 0x2 , 0x4 , 0x0 };
 
-extern const struct KeypadIcon gKeypadIcons[];
+const struct GlyphWidthFunc gGlyphWidthFuncs[] = {
+    { 0x0, GetGlyphWidthFont0 },
+    { 0x1, GetGlyphWidthFont1 },
+    { 0x2, GetGlyphWidthFont2 },
+    { 0x3, GetGlyphWidthFont3 },
+    { 0x4, GetGlyphWidthFont4 },
+    { 0x5, GetGlyphWidthFont5 },
+    { 0x6, GetGlyphWidthFont6 }
+};
 
-extern const u8 gKeypadIconTiles[];
+const struct SpriteSheet gUnknown_81EA68C[] =
+{
+    {gDoubleArrowTiles,        0x0080, 0x8000},
+    {gDoubleArrowTiles + 0x80, 0x0080, 0x8000},
```

Comment:
Looks like `gDoubleArrowTiles` is two separate files

Reply:
And then use `sizeof()` for the "size" member of` the `SpriteSheet` structs here.